docs/walkthrough.md

```diff
@@ -138,7 +138,16 @@ chmod +x ./kubectl
 We'll assume hereafter that all `kubectl` commands are using this
 newly-installed executable.
 
-To configure `kubectl` to communicate with the service catalog APU server:
+## Step 5 - Configuring `kubectl` to Talk to the API Server
+
+To configure `kubectl` to communicate with the service catalog API server, we'll have to
+get the IP address of the `Service` (and just the IP - no ports or `http://` prefixes)
```

Comment:
The instructions in this paragraph are actually a bit off. Sadly, the correct instructions are dependnet on what type of service is being used.

The default service type is `NodePort` (because the  chart is currently optimized for development and tire-kicking; not for production). As long as users haven't tweaked that, then the service IP will be a private IP in the cluster's overlay network. What they really need in this case is the IP of _the node the apiserver runs on_.

It's pretty reasonable for anyone deploying this to a real cluster _in a public cloud_ to be using service type `LoadBalancer` instead. In that case, they can pretty easily obtain the IP of the load balancer. But in that case, they shouldn't be using port `30080` as is currently indicated in the commands below.